code/modules/mob/living/carbon/xenomorph/facehuggers.dm

```diff
@@ -121,10 +121,12 @@
 
 /obj/item/clothing/mask/facehugger/examine(mob/user)
 	. = ..()
-	if(stat != CONSCIOUS)
-		to_chat(user, "<span class='warning'>[src] is not moving.</span>")
-	else
-		to_chat(user, "<span class='danger'>[src] seems to be active.</span>")
+	if(stat == CONSCIOUS)
```

Comment:
```
switch(stat)
	if(CONSCIOUS)
		stuff()
	if(UNCONSCIOUS)
		stuff()
```

Reply:
That increases more tabs

Reply:
Yea, `switch` does that by default, but the point is that `if-else` chains should be switches if they share the same common var as it's a single check instead of potentially several.
Tabs are not bad per se, else we would never have a `switch`.